weave-js/src/components/RadixTooltip/TooltipAnatomy.tsx

```diff
@@ -0,0 +1,73 @@
+import * as RadixTooltip from '@radix-ui/react-tooltip';
```

Comment:
The filename here seems a little odd, could it be RadixTooltip.tsx?

Reply:
i moved everything into index.tsx so i don't have to worry about filename :D

Reply:
Do we have a frontend style guideline on when to use a single file vs. a directory with a single index.tsx file?

Reply:
@jamie-rasmussen not that i'm aware of. do you have a suggested standard we should start doing? I've only run into this for common components and can't decide what the best method is

Reply:
My inclination would be to just have a single RadixTooltip.tsx file but not sure what the argument for the other possibility is.

Reply:
@jamie-rasmussen I was going to move the existing [`Tooltip.tsx`](https://github.com/wandb/weave/blob/bf140c4209ce39733b2dbddfc6f5d0e7fb1044c4/weave-js/src/components/Tooltip.tsx#L12) into this folder to group them all together.

I was thinking
- [ ] Replace current semantic Tooltip.tsx with Radix parts
- [ ] Rename folder from `RadixTooltip` to `Tooltip` and move the ergonomic `Tooltip.tsx` file into folder too

am i overthinking this? I could totally shove the ergonomic tooltip with the different radix parts into same file

Reply:
I see, that plan sounds reasonable to me.